http4s-blaze-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sBlazeClient.scala

```diff
@@ -0,0 +1,52 @@
+package com.avast.server.toolkit.http4s
+
+import cats.effect.{ConcurrentEffect, Resource, Sync}
+import cats.implicits._
+import cats.{Applicative, Traverse}
+import com.avast.server.toolkit.ssl.{SslContextConfig, SslContextModule}
+import javax.net.ssl.SSLContext
+import org.http4s.client.Client
+import org.http4s.client.blaze.BlazeClientBuilder
+
+import scala.concurrent.ExecutionContext
+import scala.language.higherKinds
+
+object Http4sBlazeClient {
+
+  /** Makes [[org.http4s.client.Client]] (Blaze) initialized with the given config. */
+  def make[F[_]: ConcurrentEffect](config: Http4sBlazeClientConfig, executionContext: ExecutionContext): Resource[F, Client[F]] = {
```

Comment:
I presume the `executionContext` is for callbacks, but it would be great, if it were clear from the name itself ;-)
(applies to similar occurences)

Reply:
To be honest I am not sure because the documentation is very limited. I think every `ExecutionContext` not marked specifically (and nowadays probably as `Blocker` from `cats-effect`) should be considered for callbacks. I'll document it.